### DISCUSSION TITLE
Download dashboards / settings

### DISCUSSION THREAD

**Discussion by mKlante** [agronick/aa-torque#153] [Q&A]
Hi i'm new user of AA Torque. Is there a place where i can download existing Dashbords. I'm driving Mazda 3 Skyactive X 2019. Think the demo video is using same car - would be nice to use those settings :-). 

Kind regards
Mikael

**Comment by agronick** [comment]
There is an export option in the menu. You need to make sure you export any custom PIDs from Torque as well.
